x-pack/plugins/apm/public/components/app/service_overview/service_overview_instances_chart_and_table.tsx

```diff
@@ -162,19 +157,10 @@ export function ServiceOverviewInstancesChartAndTable({
     currentPeriodItemsCount,
   } = mainStatsData;
 
-  const currentPeriodOrderedItems = orderBy(
-    // need top-level sortable fields for the managed table
-    currentPeriodItems.map((item) => ({
-      ...item,
-      latency: item.latency ?? 0,
-      throughput: item.throughput ?? 0,
-      errorRate: item.errorRate ?? 0,
-      cpuUsage: item.cpuUsage ?? 0,
-      memoryUsage: item.memoryUsage ?? 0,
-    })),
-    field,
-    direction
-  ).slice(pageIndex * PAGE_SIZE, (pageIndex + 1) * PAGE_SIZE);
+  const currentPageItems = currentPeriodItems.slice(
+    pageIndex * PAGE_SIZE,
+    (pageIndex + 1) * PAGE_SIZE
+  );
```

Comment:
If this used the new managed table, we wouldn't have to do this?

Reply:
Indeed. I'll live it for another PR though. Maybe when we add the table search to this table.